src/Compilers/Server/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -76,7 +78,8 @@ private static Assembly OnAssemblyResolve(object sender, ResolveEventArgs e)
 
         private static string GetMSBuildDirectory()
         {
-            using (var key = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\MSBuild\ToolsVersions\14.0", false))
+            var vsVersion = Environment.GetEnvironmentVariable("VisualStudioVersion") ?? "14.0";
```

Comment:
This appears to be the same as the changes in MSBuildTaskTests. Can we share the entire method?